frontend/packages/dev-console/src/components/pipelineruns/__tests__/PipelineRunDetailsPage.spec.tsx

```diff
@@ -53,7 +61,8 @@ describe('PipelineRunDetailsPage:', () => {
 
   it('Should contain task runs page', () => {
     const { pages } = wrapper.props();
-    const taskRunsPage = pages.find((page) => page.name === 'Task Runs');
+
+    const taskRunsPage = pages.find((page) => page.name === 'devconsole~Task Runs');
```

Comment:
It won't work if string is passed as variable. String should directly be passed to `t()`.